### ISSUE TITLE
Ignore quoted text in image attributes

### ISSUE THREAD

**Issue by xcoulon** [bytesparadise/libasciidoc#829]
For example, in `image::images/foo.png[*alt text*]`, the 'alt' attribute (`*alt text`) should be rendered as-is, i.e., not as bold text.
